app/client/src/components/editorComponents/ActionCreator/Fields.tsx

```diff
@@ -754,6 +819,14 @@ function renderField(props: {
         displayValue: displayValue ? displayValue : "",
       });
       break;
+    case FieldType.PAGE_NAME_AND_URL_TAB_SELECTOR_FIELD:
+      viewElement = (view as (props: TabViewProps) => JSX.Element)({
+        activeObj: props.activeNavigateToTab,
+        switches: props.navigateToSwitches,
```

Comment:
Name suggestion to keep it generic
```suggestion
       activeObj: props.activeTab,
       tabs: props.tabs,
```